src/extension/background-script/index.ts

```diff
@@ -76,7 +76,15 @@ const handleInstalled = (details) => {
 
 // listen to calls from the content script and calls the actions through the router
 // returns a promise to be handled in the content script
-const routeCalls = (message, sender) => {
+const routeCalls = (
+  message: {
```

Comment:
aren't there more attributes in this message? e.g. args?

Reply:
There are, but they are not used in this function but passed along into the router:

![image](https://user-images.githubusercontent.com/100827540/177975332-379e7b2f-b2da-4f77-a159-167dc0b37fd7.png)

There is the generic message type `Messsage` (it has been deprecated) and separate types for each message type. There is currently no common base type for messages anymore that could be used in here 🤔

Reply:
This shouldn't block this PR I think. If the deprecated type works just use it. At least files were switched to TS. That's the important step. Aligning message types can be done later. I'm happy to work on this later on.
The main goal of switching to individual message types was to clean up and identify what we currently use and also to use these in Jest tests. Maybe a Union-type works better later on.